include/subgame_menu.h

```diff
@@ -3,29 +3,76 @@
 
 #include "global.h"
 #include "task.h"
+#include "sprite.h"
+#include "data.h"
 
 struct SubGameMenu;
 
 typedef void (*SubGameMenuFunc)(struct SubGameMenu*);
 
 struct SubGameMenu {
-    u8 filler0[0x14c];
+    struct Unk_03002400 unk0[2];
+    struct Sprite unk80[5];
+    u32 unk148;
     u32 unk14C;
     u32 unk150;
     SubGameMenuFunc unk154;
-    u8 filler158[0x20];
+    u16 unk158[3][4];
+    u16 unk170;
+    u16 unk172;
+    u16 unk174;
+    u16 unk176;
     u16 unk178;
-    u8 filler17A[2];
-};
+}; /* size = 0x17C */
 
 void sub_0801E630(s32);
```

Comment:
(for files with known boundaries)
Can you find functions that are static, and mark them as static and put their forward decl in the `.c` file (if a forward decl is necessary)? Only those with external linkage should be put in the header (`.h`).